### PR TITLE
GUI: Text display issue when editing text 

### PR DESCRIPTION
**Pull request by Rohit-v-raj**

### CODE REVIEW

File: gui/widgets/editable.cpp [hunk 1]
```diff
@@ -340,7 +340,7 @@ void EditableWidget::drawCaret(bool erase) {
 		// possible glitches due to different methods used.
 		width = MIN(editRect.width() - caretOffset, width);
 		if (width > 0) {
-			g_gui.theme()->drawText(Common::Rect(x, y, x + width, y + editRect.height()), character,
+			g_gui.theme()->drawText(Common::Rect(x, y, x + width, y ), character,
```
Comment (sev-) [inline gui/widgets/editable.cpp:343]: This change does not follow our Code Formatting conventions. Please, pay attention to the spaces around braces.

**Comment by sev-** [comment]
The commit log message does not conform to our Commit Guidelines.
The code change itself breaks the Code Formatting Guidelines.

The PR is not descriptive, what is "the issue"?

**Comment by Rohit-v-raj** [comment]
When editing some text, for example in the search box, of the game name in the Edit Game dialog, the letter immediately after the text cursor is displayed lower than the rest of the text.

**Comment by criezy** [comment]
See https://wiki.scummvm.org/index.php?title=Commit_Guidelines for the commit guidelines. And pay special attention to the commit message formatting section.

You can use interactive rebase with git to fix your commit message (and squash the two commits) and then force push to your fork. This will automatically update this pull request.

**Review by criezy — CHANGES_REQUESTED** [review]
Your commit is not a proper fix for the issue.
What I suspect it does is that it no longer redraw the character when the caret is blinking (since the draw rect height is now 0). But there is a reason why we are drawing the character here after erasing the caret. What you need to find is why the character is not drawn at the correct position and fix that position.

**Comment by criezy** [comment]
@Rohit-v-raj Since there was no reply from you my request to change this pull request, and went ahead and fixed the bug myself. You can see the correct fix, and the reason of the bug in ba0dabd5c.

**Pull request closed without merging**